internal/server/server.go

```diff
@@ -372,15 +372,27 @@ func handleAllRuleStatusMetrics(rs []ruleWrapper) {
 	if conf.Config != nil && conf.Config.Basic.Prometheus {
 		var runningCount int
 		var stopCount int
+		var v promMetrics.RuleStatusMetricsValue
 		for _, r := range rs {
 			id := r.rule.Id
-			isRunning := r.state == rule.RuleStarted
-			if isRunning {
+			switch r.state {
+			case rule.RuleStarted:
 				runningCount++
-			} else {
+				v = promMetrics.RuleRunning
+			case rule.RuleStopped:
```

Comment:
You can have a case with multiple conditions to avoid code duplication. `case rule.RuleStopped, rule.RuleTerminated, rule.RuleWait:`